src/components/side_panel.scss

```diff
@@ -0,0 +1,94 @@
+%side-panel-global {
+  // Public API (allowed to be overridden)
+  --_op-side-panel-width: calc(9 * var(--op-space-x-large));
+  --_op-side-panel-header-padding: var(--op-space-medium);
+  --_op-side-panel-body-padding: var(--op-space-medium);
+  --_op-side-panel-footer-padding: var(--op-space-medium);
+  --_op-side-panel-section-padding: var(--op-space-medium);
+
+  display: flex;
+  width: var(--_op-side-panel-width);
+  min-width: var(--_op-side-panel-width);
+  height: 100vh;
```

Comment:
`height: 100%; // The 100vh is pushing things off the page.`

This assumes the panel is in something that extends fully to the bottom like a flex grow area.